docs/script.js

```diff
@@ -0,0 +1,93 @@
+const navMenuButton = document.getElementById("coffee-menu");
+const menuButton = document.getElementById("button-icon-burger");
+const navModal = document.querySelector(".nav-modal");
+const sliderContainer = document.getElementById("sliderContainer");
+let indexValue = 1;
+let slideInterval;
+let touchStartX = 0;
+let touchEndX = 0;
+
+showImage(indexValue);
+startInterval();
+
+// Touch events for slider
+sliderContainer.addEventListener("touchstart", (e) => {
+  touchStartX = e.changedTouches[0].screenX;
+});
+
+sliderContainer.addEventListener("touchmove", (e) => {
+  touchEndX = e.changedTouches[0].screenX;
+});
+
+sliderContainer.addEventListener("touchend", (e) => {
+  handleSwipe();
+});
+
+function handleSwipe() {
+  const swipeDistance = touchEndX - touchStartX;
+
+  if (Math.abs(swipeDistance) > 50) {
+    if (swipeDistance > 0) {
+      resetTime();
+      side_slide(-1);
+    } else {
+      resetTime();
+      side_slide(1);
+    }
+  }
+}
+
+function startInterval() {
+  slideInterval = setInterval(() => {
+    side_slide(1);
+  }, 5000);
+}
+
+function resetTime() {
+  clearInterval(slideInterval);
+  startInterval();
+}
+
+function btn_Slider(e) {
```

Comment:
naming convention for function names - suggesting camelCase

Reply:
Changed.